catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/GeoPdfParserImpl.java

```diff
@@ -198,7 +198,8 @@ private String getWktFromNeatLine(
 
     double[] points = new double[CTM_SIZE];
     for (int i = 0; i < CTM_SIZE; i++) {
-      points[i] = (Double) lgidict.getObjectFromPath(CTM + "/[" + i + "]").accept(toDoubleVisitor);
+      Object obj = lgidict.getObjectFromPath(CTM + "/[" + i + "]").accept(toDoubleVisitor);
+      points[i] = (obj != null) ? (Double) obj : 0d;
```

Comment:
@glenhein I saw you worked in this area of the code before. I included this check to insert a 0 in the event that the returned Object was null. Can you verify that this is a valid way to handle that possibility?